### ISSUE TITLE
Characterization of selected cluster missing

### ISSUE THREAD

**Issue by jakubklimek** [siret/similant#7]
When a cluster is selected, a list of individuals is shown. However, I have no idea what the cluster means - a characterization of that cluster (e.g. some common properties of individuals) is missing.

![image](https://user-images.githubusercontent.com/718827/53716947-e44ef480-3e56-11e9-9ad9-0ed38b1ccb47.png)

**Comment by jakubklimek** [comment]
1. We need to be able to label the properties (X-axis values) (instead/in addition to seeing their IDs). The labels can be provided in a separate file - specify the format.
1. When I click on a cluster, I need to see a word cloud (link to libraries will be added here by @skodapetr , e.g. [this](https://data.gov.cz/klíčová-slova)) of the labels

**Comment by skodapetr** [comment]
We've used the [react-tagcloud](https://www.npmjs.com/package/react-tagcloud) library for [keywords](https://data.gov.cz/kl%C3%AD%C4%8Dov%C3%A1-slova), may not be the best, but it's React and easy to use.

**Comment by dragoniscz** [comment]
It is solution, for showing just the cluster, but the app is designed for showing not only the cluster spec, but the dataset and individual spec (comparison) too. I can divide it in 3 columns, and every column will have own tagcloud (or just list).

**Comment by skodapetr** [comment]
Yes, this should show me a histogram of terms (keywords/labels) of all dataset inside a cluster.

What do you mean by the dataset and individual spec visualisation?

There is already another view (left bottom) that shows a user-provided description of the dataset.
Would this be another view but this time with the descriptors used to create a clustering?

Also what individual spec means? What is the individual?

**Comment by dragoniscz** [comment]
Same colors added for same tokens.
